pyproject.toml

```diff
@@ -17,8 +17,9 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
-    'aind-data-schema==0.38.6',
-    'pydantic<2.7',
+    'aind-data-schema==1.0.0',
+    'pydantic>=2.7',
+    'aind-data-schema-models==0.4.0',
```

Comment:
@alejoe91 For your use case, is it better to pin `aind-data-schema-models` or to just provide a lower bound?

Reply:
Lower bound is better :)